File: src/main/java/eu/mshade/enderframe/item/metadata/UnbreakableItemStackMetadata.java
package eu.mshade.enderframe.item.metadata;

import eu.mshade.enderframe.metadata.MetadataKeyValue;
import eu.mshade.enderframe.item.ItemStackMetadataKey;

public class UnbreakableItemStackMetadata extends MetadataKeyValue<Boolean> {

    public UnbreakableItemStackMetadata(Boolean metadataValue) {
        super(ItemStackMetadataKey.UNBREAKABLE, metadataValue);
    }
}

File: src/main/java/eu/mshade/enderframe/item/metadata/UnbreakableItemStackMetadata.kt
package eu.mshade.enderframe.item.metadata

import eu.mshade.enderframe.item.ItemStackMetadataKey
import eu.mshade.enderframe.metadata.MetadataKeyValue

class UnbreakableItemStackMetadata(metadataValue: Boolean) :
    MetadataKeyValue<Boolean>(ItemStackMetadataKey.UNBREAKABLE, metadataValue)
